src/stores/pacbioTubes.js

```diff
@@ -0,0 +1,50 @@
+import { defineStore } from 'pinia'
+import handlePromise from '@/api/PromiseHelper'
+import store from '@/store'
+
+export const usePacbioTubesStore = defineStore('pacbioTubes', {
+  state: () => {
+    return {
+      tractionTubes: [],
+    }
+  },
+  getters: {
+    //Pinia_migration_todo: This is migrated from the VueX store, but it needs to be changed to  Pinia store once the VueX store is removed
+    tubeRequest: () => store.state.api.traction.pacbio.tubes,
```

Comment:
We can manually make the API object here instead of using the one built from the VueX store so we dont tie any dependencies.